cinema/permissions.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
```

Comment:
There is a syntax error in the import statement. Replace the forward slash `/` with a comma `,` to correctly import `SAFE_METHODS` from `rest_framework.permissions`.